smtk-import/CMakeLists.txt

```diff
@@ -109,6 +108,13 @@ set(SMTK_LINK_LIBRARIES ${SMTK_LINK_LIBRARIES} ${SUBSURFACE_LINK_LIBRARIES} ${SS
 source_group("SmartTrak Import libs" FILES ${SMTK_IMPORT_SRCS})
 set(SMTK_IMPORT_TARGET smtk2ssrf)
 add_library(smtk_import STATIC ${SMTK_IMPORT_SRCS})
+
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
+	# Setting the link target this way resolves an issue with not importing timegettime correctly fron winmm
+	target_link_libraries(smtk_import PRIVATE wsock32 ws2_32 ole32 imm32 winmm)
```

Comment:
timeGetTime() comes from winmm, i vaguely remember issues around ordering of link libraries, but can you explain what happens here?

Reply:
With the latest QT version, running with the original link libraries line the app complains about missing symbol.  I tried reordering things but it didn't seem to help. I beleive this is caused by the updated cmake changing how it handles transitive dependancies. This seemed to be the only way I found to resolve issue.

Reply:
ok. yes, reordering libraries to put the problem one last solves this from my experience.